dvc/cache.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, repo):
         self.local = Remote(repo, settings)
         self.s3 = self._get_remote(config, Config.SECTION_CACHE_S3)
         self.gs = self._get_remote(config, Config.SECTION_CACHE_GS)
+        self.oss = self._get_remote(config, Config.SECTION_CACHE_OSS)
```

Comment:
Not needed, since we don't support external dependencies/outputs on OSS. Azure shouldn't have cache for those same reasons too, but there are some leftovers from previous experiments, we'll remove those too. See related adjustments down below.